### PR TITLE
Updated vocab doc - highly desirable table is still to update

### PR DESCRIPTION
**Pull request by emmerbodc**
I've made some large changes to split out the mandatory and high desirable metadata fields. 
Also added some text about requesting new terms. We need to decide on labels for new vocab issues.

Proponents: 
Moderator: @OceanGlidersCommunity/format-mantainers

# Type of PR

- [ ] Typo without possible change of interpretation of the related text.
- [ ] Fix of some error, inconsistency, unforeseen limitation.
- [ ] Style that only affects visually the compiled document
- [ ] Addition that does not require change in the current structure.
- [ ] Enhancement that require changes to improve the format.

# Related Issues

# Dates when it got review approvals

# Release checklist

- [ ] Approved by at least two members of the committee?
- [ ] There were modifications after the review approvals? If so, please
      ask reviewers to update their review.
- [ ] Proponents and moderador should explicitly agree that it is ready to
      to merge.
- [ ] The moderador is the one in charge to actually merge or close this PR
      according to the final decision.

# For maintainers

- [ ] Update the moderator with a volunteer from the committee. It would be
      best to have one single moderator to guide and help this PR to move
      forward. It is OK to update the moderador pass it to another one.
- [ ] Confirm that the associated branch was deleted after the merging.
- [ ] Wrap-up and close the related issues.

# Comments

### CODE REVIEW

**Comment by emmerbodc** [comment]
@vturpin  updated vocab document - mainly creating a new table for only mandatory terms.

**Review by callumrollo — APPROVED** [review]

File: vocabularyCollection/tableOfControlledVocab.adoc [hunk 1]
```diff
@@ -28,52 +28,72 @@ Vocabulary collections that are not governed by OceanGliders do not follow the *
...
 
-=== Request a new entry
-**TO BE UPDATED WITH EMMA and NVS TEAM**
+=== Requesting a new term
+To request a new term for an existing vocabularly collection please create an new issue in the OceanGliders repository including the label 'vocab'.
+
+New terms may also be requested via the 'Request new term' URLs in the table below. Please clearly indicate in the ticket 'for approval by OceanGliders vocabularly working group' and include as much detail as possible. 
+
+=== Requesting a new collection
+If you are unsure where a new term fits and believe a new collection is needed, please create an an issue in the OceanGliders repository clearly including the title 'vocab'.
```
Comment (castelao) [inline vocabularyCollection/tableOfControlledVocab.adoc:37]: Typo: 'an an'

**Review by castelao — COMMENTED** [review]
Looks great. I think this change would be very helpful.

**Review by vturpin — APPROVED** [review]

**Review by vturpin — APPROVED** [review]

File: vocabularyCollection/tableOfControlledVocab.adoc (hunk 1 above)
Comment (vturpin) [reply vocabularyCollection/tableOfControlledVocab.adoc]: Correction made by myself

**Pull request merged**